src/main/java/personList/ListActions.java

```diff
@@ -0,0 +1,70 @@
+package personList;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class ListActions {
+    public static boolean contains(List<Person> personList, Person person) {
+        boolean containsResult = false;
+        PersonsComparator comparePersons = new PersonsComparator();
+        for (Person e : personList) {
+            if (comparePersons.compare(e, person) == 0) {
+                containsResult = true;
+            }
+        }
+        return containsResult;
+    }
+
+    public static boolean equalLists(List<Person> personList1, List<Person> personList2) {
+        Iterator<Person> iter = personList1.iterator();
+        boolean containsResult = false;
+        if (personList1.size() != personList2.size()) {
+            containsResult = false;
+        } else {
+            while (iter.hasNext()) {
+                Person next = iter.next();
+                if (!contains(personList2, next)) {
+                    containsResult = false;
+                } else {
+                    containsResult = true;
+                }
+            }
+        }
+        return containsResult;
+    }
+
+    public static void printOut(List<Person> personList1) {
+        for (Person p : personList1) {
+            System.out.println(p);
+        }
+    }
+
+    public static List<Person> uniquePersons(List<Person> personList) {
+        List<Person> uniqueList = new ArrayList<>();
+        Iterator<Person> iter = personList.iterator();
+        while (iter.hasNext()) {
+            Person next = iter.next();
+            if (!contains(uniqueList, next)) {
+                uniqueList.add(next);
+            }
+        }
+        return uniqueList;
+    }
+
+    public static List<Person> sort2(List<Person> personList) {
```

Comment:
As for method design, if you modify an existing list, so no need to return value in this method, since all changes will be reflected outside of this method. 
If you want your method to return value, so it's better to create a new list and make it sorted while original one should not be touched

Reply:
Removed return.